### PR TITLE
chore(linux): Remove package build on Jenkins for Keyman 17

### PR DESCRIPTION
**Pull request by ermshiperete**
@keymanapp-test-bot skip

### CODE REVIEW

File: resources/build/build-utils.sh [hunk 1]
```diff
@@ -74,7 +74,7 @@ function findVersion() {
         VERSION_TAG=
     fi
 
-    if [ -z "${TEAMCITY_VERSION-}" -a -z "${JENKINS_HOME-}" ]; then
+    if [ -z "${TEAMCITY_VERSION-}" ]; then
```
Comment (mcdurdin) [inline resources/build/build-utils.sh:77]: Do we need a GHA condition here?

**Review by mcdurdin — APPROVED** [review]
LGTM

File: resources/build/build-utils.sh (hunk 1 above)
Comment (ermshiperete) [reply resources/build/build-utils.sh:77]: → https://github.com/keymanapp/keyman/pull/9400

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.152-alpha](https://keyman.com/downloads/releases/alpha/17.0.152)

**Pull request merged**